helpers/discovery/editDiscovery.ts

```diff
@@ -30,6 +30,7 @@ export async function editDiscovery(bot: Bot, guildId: bigint, data: ModifyGuild
 }
 
 // TODO: add docs link
+/** https://github.com/discord/discord-api-docs/pull/1762 */
```

Comment:
Please remove the `TODO` comment

Reply:
@itohatweb check out https://github.com/discord/discord-api-docs/pull/1762#issuecomment-1082095053
the document discovery pr is closed not merged
I just want to put the link to the PR so the next person wont have to waste half an hour searching discord docs and cant find discovery stuff

Reply:
Then remove this reference.